File: bank.java
import java.util.Scanner;

public class bank {
    int acc_no;
    String name, acc_type;
    float bal;
    Scanner ob = new Scanner(System.in);

    void getValues() {
        System.out.print("Enter account number:");
        acc_no = ob.nextInt();
        System.out.print("Enter name:");
        name = ob.nextLine();
        System.out.print("Enter account type:");
        acc_type = ob.nextLine();
        System.out.print("Enter balance:");
        bal = ob.nextFloat();
    }

    void deposit() {
        int amount;
        System.out.println("\t"+"DEPOSIT PROCESS"+"\t");
        System.out.print("Amount to be deposited:");
        amount = ob.nextInt();
        bal += amount;
        System.out.println("Current balance: " + bal);
       
    }

    void withdraw() {
        int amount;
        System.out.println("\t"+"WITHDRAW PROCESS"+"\t");
        System.out.println("Amount to be withdrawn:");
        amount = ob.nextInt();
        if(bal>0)
        {
            bal -= amount;
            System.out.print("Current balance: " + bal);
        }
        else
        {
            System.out.print("No balance: ");
        }
    }

    void display() {
        System.out.println("\t"+"DETAILS"+"\t");
        System.out.println("Account number: " + acc_no);
        System.out.println("Name: " + name);
        System.out.println("Account type: " + acc_type);
        System.out.println("Current balance: " + bal);
    }

    public static void main(String args[]) {
    
        bank b = new bank();
        b.getValues();
        b.deposit();
        b.withdraw();
        b.display();
            
        } 
       
    }

File: bank.py
# -*- coding: utf-8 -*-
"""
Created on Sun Jan 19 18:14:00 2025

@author: Madhumitha
"""

class Bank:
    def __init__(self):
        self.current_amount=0
        
    def getvalues(self):
        print("------- ENTER YOUR DETAILS ----------")
        self.name=input("Enter Name:")
        self.acc_num=int(input("Enter Account Number:"))
        self.acc_type=input("Enter Account Type:")


    def deposit(self):
        self.deposit_amount=int(input("Enter Deposit Amount:"))
        self.current_amount+=self.deposit_amount
        

    def withdraw(self):
        self.withdraw_amount=int(input("Enter Withdraw Amount:"))
        if(self.withdraw_amount<self.current_amount):
            self.current_amount-=self.withdraw_amount
        else:
            print("Your balance is low!")
        
    def show_balance(self):
        self.current_amount=self.current_amount
        print("Current Balance:",self.current_amount) 
        
        
    def display(self):
        print("------------DETAILS--------------")
        print("Name:",self.name)   
        print("Account-Number:",self.acc_num)   
        print("Account-Type:",self.acc_type)   
        print("Current Balance:",self.current_amount)  
        
        
b=Bank()

while(True):
    print("-------BANK APPLICATION----------")
    print("1. Getting Details")
    print("2. Deposit Amount")
    print("3. Withdraw Amount")
    print("4. Displaying Details")
    print("5. Show Current Balance")
    print("6. Quit")

    ch=int(input("Enter your choice:"))
    if(ch==1):
        b.getvalues()
    elif(ch==2):
        b.deposit()
    elif(ch==3):
        b.withdraw()
    elif(ch==4):
        b.display()   
    elif(ch==5):
        b.show_balance()   
    elif(ch==6):
        print("Thank you and have a nice day!")
        break
    else:
        print("Invalid Option")
      